README.md

```diff
@@ -34,30 +38,39 @@ The server component to the TourTrak system built using Django.
 * Setup the postGIS functions:
 
 ```
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/postgis.sql
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/spatial_ref_sys.sql
-  
+  psql DCS
+  CREATE EXTENSION postgis;
+  CREATE EXTENSION postgis-topology;
```

Comment:
You have a typo here. This should be

    CREATE EXTENSION postgis_topology;

(note the underscore vs the dash)